model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -463,8 +518,8 @@ object ArrowInliner extends Logging {
       Arrows[S].scope(
         for {
           // Process renamings, prepare environment
-          fn <- ArrowInliner.prelude(arrow, call, exports, arrows)
-          inlineResult <- ArrowInliner.inline(fn, call, streams)
+          fnCanon <- ArrowInliner.prelude(arrow, call, exports, arrows)
+          inlineResult <- ArrowInliner.inline(fnCanon._1, call, streams, fnCanon._2)
         } yield inlineResult
```

Comment:
Can we unite `canon`s returned from `prelude` on this level? Passing it to `inline` adds complexety for nothing

Reply:
`canon`s here is a model, because we handling VarModel, but `body` is Raw, because we didn't handle it. I can create `VarRaw` from `VarModel`, but it is not ok

Reply:
Yes, but no need to unite with `body`. It is possible to add `canon`s to `InlineResult`.